packages/jest-worker/src/worker.js

```diff
@@ -49,12 +49,13 @@ export default class {
   _busy: boolean;
   _child: ChildProcess;
   _options: WorkerOptions;
-  _queue: Array<QueueChildMessage>;
+  _queue: ?QueueChildMessage;
+  _last: ?QueueChildMessage;
```

Comment:
this is not in the right alphabetical location :P

Reply:
😄